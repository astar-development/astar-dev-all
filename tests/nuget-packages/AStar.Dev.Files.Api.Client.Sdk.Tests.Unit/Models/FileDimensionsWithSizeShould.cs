using AStar.Dev.Files.Api.Client.SDK.Models;

namespace AStar.Dev.Files.Api.Client.Sdk.Models;

public sealed class FileDimensionsWithSizeShould
{
    [Fact]
    public void ReturnTheExpectedToString() =>
        new FileDimensionsWithSize().ToString().ShouldMatchApproved();
}
