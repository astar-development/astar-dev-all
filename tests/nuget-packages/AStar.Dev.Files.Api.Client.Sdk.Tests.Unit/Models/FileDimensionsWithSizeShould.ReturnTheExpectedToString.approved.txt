{
  "fileLength": 0,
  "height": 0,
  "width": 0,
  "fileSizeForDisplay": "0.00 Kb"
}